_posts/2023-09-15-kafka-producer-client-essentials.md

```diff
@@ -296,11 +291,14 @@ You might also want to explore how to expand and improve on other aspects of you
 Implement security to establish a secure connection using authentication and authorization mechanisms when connecting to the Kafka cluster. 
 For example, you can set up TLS authentication for external clients in your Strimzi environment and add the TLS certificates to your client configuration. 
 
+You can use configuration providers to load configuration, including secrets, from external sources.
```

Comment:
As an example, I would mention specifically the `KubernetesSecretConfigProvider` to extract TLS certificates and keys directly from the Kubernetes API. This is handy, because you won't need to do any configuration change, just a simple application restart (hot reloading is not supported right now).